ftplugin/ledger.vim

```diff
@@ -508,6 +508,8 @@ command! -buffer -nargs=+ -complete=customlist,s:autocomplete_account_or_payee
 
 command! -buffer -range LedgerAlign <line1>,<line2>call ledger#align_commodity()
 
+command! -buffer LedgerAlignAll call ledger#align_all_commodity()
```

Comment:
```suggestion
command! -buffer LedgerAlignBuffer call ledger#align_commodity_buffer()
```

Is there a precedent for `...All` functions that act on the entire buffer or would this be more explicit? In any case the word order should be the same for the internal command.

Reply:
Resolved in 0fc0d2e.

Reply:
> Is there a precedent for ...All functions that act on the entire buffer or would this be more explicit? 

Not that I'm particularly aware of, although I should also note that I haven't really delved into contributing to rules in the past so my experience is limited. I liked your suggestion of `LedgerAlignBuffer`, though.